Adding the requirements document & putting a link to it in the README

l00144427 opened the pull request:
Renamed the document to requirements.md & put a link to it in the README.md file

l00144427 committed:
Adding the requirements document & putting a link to it in the README

rlennon commented on content/requirements.md:
```diff
@@ -0,0 +1,69 @@
+
+* Branch location should be entered.
+
+* Dimensions can be entered.
```
Additional information would be useful such as the number of rooms, special room requirements and so on

l00144427 replied:
I have added some changes based on your comments, please review & approve if you think it's ok.

rlennon commented on content/requirements.md:
```diff
@@ -0,0 +1,69 @@
+
+Do we need to worry about adding & deleting users?
+
+Do we need to worry about adding & deleting branches?
```
Yes

rlennon commented on content/requirements.md:
```diff
@@ -0,0 +1,69 @@
+
+**Outstanding Questions**
+
+Do we need to worry about adding & deleting users?
```
yes

l00144427 committed:
Added additional information regarding the branches and users as requested in the peer review

l00144427 committed:
Merge branch 'master' into emmet-branch

l00144427 commented:
I added additional information to the Users, Data Entry Requirements Details & Other Details sections and also removed the Outstanding Questions section based on the feedback.
Please review and approve if acceptable.

cjfarrelly reviewed (approved):
lgtm!

l00144427 committed:
Merge branch 'master' into emmet-branch

L00144438 reviewed (approved):
Need to discuss the requirements!

l00073976 reviewed (approved):
lgtm

rlennon reviewed (approved):
LGTM
